src/app/dashboard-widgets/work-item-widget/work-item-widget.component.ts

```diff
@@ -83,9 +83,15 @@ export class WorkItemWidgetComponent implements OnInit {
           100000, {expression: {'space': `${space.id}`}}
         );
       })
+      .do((val: { workItems: WorkItem[], nextLink: string, totalCount?: number, included?: WorkItem[], ancestorIDs?: string[]}): void => {
```

Comment:
Does `WorkItemService` export a definition for this type? If not, I'd like to see this lifted out from this inline definition to the file top-level, or maybe at the class level. I would also probably open an issue against Planner to export a definition for consumers to use.